templates/daemon.json.j2

```diff
@@ -1,6 +1,11 @@
 {
+{% if docker.thin_pool_device is not none %}
   "storage-driver": "devicemapper",
   "storage-opts": [
     "dm.directlvm_device={{ docker.thin_pool_device }}"
-  ]
+  ]{% if docker.bridge_ip is not none %},{% endif %}
+{% endif %}
+{% if docker.bridge_ip is not none %}
+  "bip": "{{ docker.bridge_ip }}"
+{% endif %}
 }
```

Comment:
Just double checking, the service doesn't care if this file is just empty JSON? I'm sure it's fine but it's best to be sure :)

Reply:
Ive tested this and it showed that docker is happy with a config file containing "{}"